.travis.yml

```diff
@@ -3,10 +3,13 @@ cache:
   directories:
     - node_modules
 node_js:
-  - 4.2
+  - 6.10.2
```

Comment:
I think we should support Node 4 since that is the current target for Roc itself.